smartsim/settings/batchSettings.py

```diff
@@ -44,12 +44,73 @@
 
 
 class BatchSettings(BaseSettings):
+    """The BatchSettings class manages the configuration and execution of batch jobs
```

Comment:
Maybe this top line comment would be better as:

The BatchSettings class stiores scheduler configuration settings and is used  
    inject scehduler-specific behavior into a batch job.